File: contest/LeetCode_Weekly_Contest_202/Three_Consecutive_Odds.py
class Solution(object):
    def threeConsecutiveOdds(self, a):
        """
        :type arr: List[int]
        :rtype: bool
        """
        return any(map(lambda st: a[st]&1 and a[st+1]&1 and a[st+2]&1, [i for i in xrange(len(a)-2)]))

File: contest/LeetCode_Weekly_Contest_202/Three_Consecutive_Odds.java
class Solution {
    public boolean threeConsecutiveOdds(int[] arr) {
        int n = arr.length;
        for (int i = 0; i+3 <= n; ++i) {
            if ((arr[i]%2 == 1) && (arr[i+1]%2==1) && (arr[i+2]%2==1))
                return true;
        }
        return false;
    }
}
